src/serac/numerics/functional/domain_integral_kernels.hpp

```diff
@@ -265,14 +268,14 @@ void action_of_gradient_kernel(const double* dU, double* dR, derivatives_type* q
   using test_element  = finite_element<g, test>;
   using trial_element = finite_element<g, trial>;
 
-  static constexpr bool is_QOI   = (test::family == Family::QOI);
-  static constexpr int  num_qpts = num_quadrature_points(g, Q);
+  const bool is_QOI   = (test::family == Family::QOI);
```

Comment:
Why are these no longer `constexpr`?

Reply:
I changed them back.  They cannot be static and be captured by device lambda but can be constexpr